docs/tour/why_wasmcloud.mdx

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 3
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Hello Universe
+
+You've now built a simple microservice with wasmCloud that has common capabilities (handle HTTP requests, interface with a Key-Value store) that many cloud native applications need. So, why wasmCloud? What makes this application that you built different from similar apps on other platforms? Beyond providing an easy-to-use platform that's quick to get started with, we also want to make sure that you understand that the biggest benefits of wasmCloud come after you've built your application.
```

Comment:
change "has common capabilities" to something like "uses" or "leverages common capabilities"

Reply:
"So, why wasmCloud?" -> So why wasmCloud? , maybe italicize _why wasmCloud_